### PR TITLE
Updating page title bar

### PR DESCRIPTION
**Pull request by line47**
Moved title bar up into the header container. Passed a property to main container for page title as the main container is the only container that know the header exists. Created a container for the content as the title bar for the application is more complex showing status and top level application actions.

screen shot 2016-11-30 at 1 23 07 pm

### CODE REVIEW

**Comment by vz3** [comment]
I'll take a swing at this review

**Comment by line47** [comment]
I need to do a PR for cg-style related to these changes.

**Comment by line47** [comment]
Relies on https://github.com/18F/cg-style/pull/198

File: static_src/components/app_container.jsx [hunk 1]
```diff
@@ -128,8 +128,8 @@ export default class AppContainer extends React.Component {
     } else if (!this.state.loading && appReady(this.state.app)) {
       content = (
         <div>
-          <h2>{ this.fullTitle } { this.statusUI }</h2>
-          { this.restart }
+
```
Comment (msecret) [inline static_src/components/app_container.jsx]: These extra lines should be removed?

File: static_src/components/service_list.jsx [hunk 2]
```diff
@@ -55,6 +55,7 @@ export default class ServiceList extends React.Component {
     } else if (this.state.services.length) {
       content = (
       <div>
+
```
Comment (msecret) [inline static_src/components/service_list.jsx]: I think this newline should be removed

File: static_src/components/app_container.jsx (hunk 1 above)
Comment (line47) [inline static_src/components/app_container.jsx]: I'll remove them.

File: static_src/components/service_list.jsx (hunk 2 above)
Comment (line47) [inline static_src/components/service_list.jsx]: I'll remove it.

**Comment by line47** [comment]
I'm good with updating the other page titles as well before merging.

**Pull request closed without merging**